### ISSUE TITLE
--why and multiple entries

### ISSUE THREAD

**Issue by theKashey** [ai/size-limit#117]
There two issues with `--why`
- it does not support multiple entries, showing results only for the first one. I would like to be able to specify an entry point to "explain why"
- it does not do the work well.

### Let me elaborate on this
First of all - what is the goal of `--why`? Explain "what's inside, and why it's so big". Does it work? Nooooopppeee, as long as you are size-limiting __esm__(aka `modules:false`) bundle, which got properly tree shaked(👍), and concatenated(👎), with all the details lost.
- [ ] disable concatenation for `--why`

Secondary, as I asked above, I want to _size-limit_ multiple entries, and I would like to call them by names, or even just use these entry points as `path`
```diff
  {
-    path: "dist/es2015/index.js",
+    path: ".", // look for package.json/index.js -> resolve target file -> do the jon
+    path: "stuff", // look for stuff.js, stuff/package.json, stuff/index.js and so on
     limit: "2.0 KB"
  },
```
The "file" behind `path` could be resolved via `Module._resolveFilename`

### Outcome
You will measure not you what want, but what user will consume,

- [ ] WYSIWYG size-limit. Yep, it's really "what you see is what you get"

**Comment by ai** [comment]
> Nooooopppeee, as long as you are size-limiting esm(aka modules:false) bundle, which got properly tree shaked(+1), and concatenated(-1), with all the details lost.

Can you give me more details?

Do we have some Webpack Bundle Analyzer issue with modules?

> it does not support multiple entries, showing results only for the first one.

What is your current config and `--why` output?

> I would like to call them by names, or even just use these entry points as path

Did you saw `name` config entry? https://github.com/ai/size-limit#config

**Comment by theKashey** [comment]
> Did you saw name config entry? https://github.com/ai/size-limit#config

That's different. I am asking for another "name", I want to name something in a same way, external customer would do - `my-lib`, `my-lib/stuff` and so on. For example `loadable-components` are using rollup generated `min.js` to track their size, while expose `cjs/esm` for the final consumer. Well - that's not quite fair.

> Do we have some Webpack Bundle Analyzer issue with modules?

No, that's bound to the default webpack optimisation - module concatenation. Here is a example (using files from cjs/esm bundles) - one is detailed, another is `XXX files concatenated`:
<img width="1508" alt="Screen Shot 2019-06-09 at 6 48 53 pm" src="https://user-images.githubusercontent.com/582410/59157023-5f274b80-8ae7-11e9-9650-04f09b944069.png">
<img width="1505" alt="Screen Shot 2019-06-09 at 6 48 39 pm" src="https://user-images.githubusercontent.com/582410/59157024-5fbfe200-8ae7-11e9-877b-eea075a79f0b.png">

while module concatenation should be used to used to calculate bundle size - the "why" part should not use it.

**Comment by ai** [comment]
Можешь объяснить по русски? Что-то я не понимаю что требуется

**Comment by ai** [comment]
Released in 1.3.7
